osm2gtfs.py

```diff
@@ -298,5 +293,34 @@ def load_config(file):
 
     return config;
 
+def load_agency(config):
+    '''
+    Loads agency data from a json config file. The valid keys under the json
+    "agency" object correspond to the transitfeed.Agency._FIELD_NAMES.
+
+    return a transitfeed.Agency object
+'''
+
+    data_dict = {}
+    for field_name in transitfeed.Agency._FIELD_NAMES:
```

Comment:
This variable is protected, we should not access it directly. I'll change this before merging.